opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -361,7 +369,6 @@ class Translator {
             .catch(function (err) {
                 console.log("Request failed to: " + options.method + " - " + options.url);
                 console.log("Error            : " + err.statusCode + " - " + err.response.statusMessage);
-                // todo auto refresh in specific cases, issue 74
```

Comment:
[https://github.com/openT2T/translators/issues/74](https://github.com/openT2T/translators/issues/74)

we can't do auto-refresh, I'm going to close the issue.  We need an explicit call to do refresh.

---
In reply to: [88992135](https://github.com/openT2T/translators/pull/126#discussion_r88992135) [](ancestors = 88992135)